plugins/command-line/prism-command-line.js

```diff
@@ -35,41 +96,30 @@ Prism.hooks.add('complete', function (env) {
 	}
 
 	// Create the "rows" that will become the command-line prompts. -- cwells
-	var lines = new Array(1 + env.code.split('\n').length);
+	var promptLines = new Array(1 + env.code.split('\n').length);
```

Comment:
It seems a bit wasteful to split `env.code` again, just to get to the number of lines. You could store that information in the `env` object and reuse it here.